demos/kdf/hkdf.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2021-2023 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 2021-2024 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
Hmm, it doesn't look to me that this file was updated in 2024 - the last change I see in here is

```
commit 86db958835d1f8ba9ce49a9f93b5309c3d13b91c
Author: James Muir <james@openssl.org>
Date:   Fri Nov 10 14:02:00 2023 -0500

    demos: tidy up makefiles, fix warnings
```